addon/run-task.ts

```diff
@@ -229,6 +229,7 @@ export function throttleTask(
   let timers: Set<EmberRunTimer> = getTimers(destroyable);
   let cancelId: EmberRunTimer = throttle(
     destroyable as any,
+    // @ts-ignore
```

Comment:
This was to avoid the type mismatch in throttle.

Reply:
ya, makes sense (might be good to throw a comment in there so we remember)

Reply:
Will do.